plasmapy/physics/parameters.py

```diff
@@ -178,12 +179,13 @@ def Alfven_speed(B, density, ion="p"):
     except Exception:
         raise ValueError("Unable to find Alfven speed")
 
-    _check_relativistic(V_A, 'Alfven_speed')
-
     return V_A
 
 
-def ion_sound_speed(*ignore, T_e=0*units.K, T_i=0*units.K,
+@check_relativistic
+@check_quantity("T_e", can_be_negative=False)
+@check_quantity("T_i", can_be_negative=False)
```

Comment:
Case 1:
```python
@check_quantity({
    "T_e": { "units": units.K, "can_be_negative": False},
    "T_i": { "units": units.K, "can_be_negative": False},
    "B": {"units": units.T}
})
def func(T_e, T_i, B):
    pass
```
Case 2:
```python
@check_quantity({
    "T_e": { "can_be_negative": False},
    "T_i": { "can_be_negative": False}
})
def func(T_e:  units.K, T_i: units.K, B: units.T):
    pass
```
It all comes down to a style choice. Which one do you prefer?